app/src/main/java/com/duckduckgo/app/trackerdetection/api/TrackerDataDownloader.kt

```diff
@@ -57,8 +57,9 @@ class TrackerDataDownloader @Inject constructor(
 
             Timber.d("Updating tds data from server")
             val body = response.body()!!
+            val eTag = response.headers()["eTag"]!!.removeSurrounding("W/\"", "\"") // removes weak eTag validator
```

Comment:
is the `eTag` guaranteed to be there? should we play safe as treat it as nullable?

Reply:
In theory it should be but because we don't trust theories I'll return an empty string just in case.